fusor-ember-cli/app/routes/storage.js

```diff
@@ -1,6 +1,10 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  setupController(controller, model) {
```

Comment:
This is actually the default behavior for setupController, so it's unnecessary to override, you can safely remove it.

http://emberjs.com/api/classes/Ember.Route.html#method_setupController if you're curious